### ISSUE TITLE
Using another parameter settings in the request

### ISSUE THREAD

**Issue by eduavila** [prettus/l5-repository#20]
How can I use for other parameter settings in RequestCriteria

database column = da3_cod

requestCriteria

my params
?cod=2030

It is possible to change the parameter settings.

l5-repository compatible with Laravel 4.2?

**Comment by andersao** [comment]
@eduavila 

I will release a new version, it you will be able to search as follows:

`?search=VALUE_SEARCHED`

`?search=FIELD:VALUE_SEARCHED`

`?search=FIELD:VALUE_SEARCHED;ANOTHER_FIELD:VALUE_SEARCHED`

`?search=FIELD:VALUE_SEARCHED;VALUE_SEARCHED_IN_ANOTHER_FIELDS`

`?search=VALUE_SEARCHED;searchFields:FIELD:CONDITION`

In your case, you can configure the fields that are searchable in your repository so

```php
use Prettus\Repository\Eloquent\BaseRepository;
use Prettus\Repository\Criteria\RequestCriteria;


class MyRepository extends BaseRepository {

    /**
     * @var array
     */
    protected $fieldSearchable = [
        'description'=>'like',
        'cod'
    ];

    public function boot(){
        $this->pushCriteria(app('Prettus\Repository\Criteria\RequestCriteria'));
        ...
    }

    function model(){
       return "App\\MyModel";
    }
}
```

If you perform the following request

`?search=VALUE`

There will be equivalent

```sql
select * from `your_table` where `description` like '%VALUE%' or `cod` = 'VALUE';
```

`?search=VALUE&searchFields=cod` or `?search=cod:VALUE`

There will be equivalent

```sql
select * from `your_table` where `cod` = 'VALUE';
```

```sql
?search=description:VALUE;cod:VALUE
```

There will be equivalent

```sql
select * from `your_table` where `cod` = 'VALUE';
```

Well, hope this helps

**Comment by andersao** [comment]
Released version 2.0.6 :+1:
